scalafix-core/shared/src/main/scala/scalafix/patch/Patch.scala

```diff
@@ -48,12 +48,16 @@ sealed abstract class Patch {
   def ++(other: Iterable[Patch]): Patch = other.foldLeft(this)(_ + _)
   def isEmpty: Boolean = this == EmptyPatch
   def nonEmpty: Boolean = !isEmpty
+  /** Skip this entire patch if a part of it is disabled with // scalafix:off
```

Comment:
Nitpick, single line docstrings should be `/** Blah blah */`, I should make scalafmt enforce this 😅